lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -1005,27 +998,26 @@ def _AssignCpuAffinity(cls, cpu_mask, process_id, thread_dict):
       for i, vcpu in enumerate(cpu_list):
         cls._SetProcessAffinity(thread_dict[i], vcpu)
 
-  def _GetVcpuThreadIds(self, instance_name):
+  @_with_qmp
+  def _GetVcpuThreadIds(self, instance):
     """Get a mapping of vCPU no. to thread IDs for the instance
 
     @type instance_name: string
```

Comment:
The argument type changes as well.

Reply:
Yes, will be fixed. Missed that in different place as well.